packages/contracts-bedrock/test/OptimismPortal.t.sol

```diff
@@ -135,199 +144,357 @@ contract OptimismPortal_Test is Portal_Initializer {
     /// @dev Tests that `depositTransaction` reverts when the gas limit is too small.
     function test_depositTransaction_smallGasLimit_reverts() external {
         vm.expectRevert("OptimismPortal: gas limit too small");
-        op.depositTransaction({ _to: address(1), _value: 0, _gasLimit: 0, _isCreation: false, _data: hex"" });
+        optimismPortal.depositTransaction({ _to: address(1), _value: 0, _gasLimit: 0, _isCreation: false, _data: hex"" });
     }
 
     /// @dev Tests that `depositTransaction` succeeds for small,
     ///      but sufficient, gas limits.
     function testFuzz_depositTransaction_smallGasLimit_succeeds(bytes memory _data, bool _shouldFail) external {
         vm.assume(_data.length <= type(uint64).max);
 
-        uint64 gasLimit = op.minimumGasLimit(uint64(_data.length));
+        uint64 gasLimit = optimismPortal.minimumGasLimit(uint64(_data.length));
         if (_shouldFail) {
             gasLimit = uint64(bound(gasLimit, 0, gasLimit - 1));
             vm.expectRevert("OptimismPortal: gas limit too small");
         }
 
-        op.depositTransaction({ _to: address(0x40), _value: 0, _gasLimit: gasLimit, _isCreation: false, _data: _data });
+        optimismPortal.depositTransaction({
+            _to: address(0x40),
+            _value: 0,
+            _gasLimit: gasLimit,
+            _isCreation: false,
+            _data: _data
+        });
     }
 
     /// @dev Tests that `minimumGasLimit` succeeds for small calldata sizes.
     ///      The gas limit should be 21k for 0 calldata and increase linearly
     ///      for larger calldata sizes.
     function test_minimumGasLimit_succeeds() external {
-        assertEq(op.minimumGasLimit(0), 21_000);
-        assertTrue(op.minimumGasLimit(2) > op.minimumGasLimit(1));
-        assertTrue(op.minimumGasLimit(3) > op.minimumGasLimit(2));
+        assertEq(optimismPortal.minimumGasLimit(0), 21_000);
+        assertTrue(optimismPortal.minimumGasLimit(2) > optimismPortal.minimumGasLimit(1));
+        assertTrue(optimismPortal.minimumGasLimit(3) > optimismPortal.minimumGasLimit(2));
     }
 
     /// @dev Tests that `depositTransaction` succeeds for an EOA depositing a tx with 0 value.
-    function test_depositTransaction_noValueEOA_succeeds() external {
+    function testFuzz_depositTransaction_noValueEOA_succeeds(
+        address _to,
+        uint64 _gasLimit,
+        bytes memory _data
+    )
+        external
+    {
+        vm.assume(_gasLimit < systemConfig.gasLimit());
+        vm.assume(_data.length < type(uint64).max);
+        vm.assume(_gasLimit > optimismPortal.minimumGasLimit(uint64(_data.length)));
+        vm.assume(_gasLimit < systemConfig.resourceConfig().maxResourceLimit);
```

Comment:
This file has lots of assume usage which can slow down tests, so I typically prefer `bound` and would recommend that here. Can also remove `vm.assume(_data.length < type(uint64).max)` since forge will never generate a value of that many bytes (would run out of memory if it tried since that's so large)

Reply:
Ok i can switch to bound, good call